common/account_utils/src/validator_definitions.rs

```diff
@@ -113,18 +128,27 @@ impl ValidatorDefinition {
 
 /// A list of `ValidatorDefinition` that serves as a serde-able configuration file which defines a
 /// list of validators to be initialized by this validator client.
-#[derive(Default, Serialize, Deserialize)]
-pub struct ValidatorDefinitions(Vec<ValidatorDefinition>);
+pub struct ValidatorDefinitions {
+    definitions: Vec<ValidatorDefinition>,
+    _lockfile: Lockfile,
+}
 
 impl ValidatorDefinitions {
+    pub fn create<P: AsRef<Path>>(validators_dir: P) -> Result<Self, Error> {
+        Ok(Self {
+            definitions: vec![],
+            _lockfile: Lockfile::new(validators_dir.as_ref().to_path_buf())?,
```

Comment:
Getting close!

I think this should join the `LOCK_FILE` suffix

Reply:
Consult 19cfe3b5.